server/cache.js

```diff
@@ -17,6 +18,11 @@ module.exports = Cache;
 
 function Cache(ecs){
   Emitter.call(this);
+  this.tasks = LRU({
+    max: 1000000,
+    maxAge: ms('1d'),
```

Comment:
An entire day seems like a lot. Do we really need to keep this stuff cached for so long?